plugin.rb

```diff
@@ -191,4 +191,13 @@
 
     false
   end
+
+  register_category_custom_field_type('create_as_qa_default', :boolean)
```

Comment:
I think we should use constant here similar to how we did in [chat plugin](https://github.com/discourse/discourse-chat/blob/9c4271b846aa03d7bce6e553dc068043e08c9e9c/plugin.rb#L203) and then reuse that constant.

Reply:
Added https://github.com/discourse/discourse-question-answer/pull/96/commits/dc72d4456422ba58a4472b059fbb902efc0264b7